### PR TITLE
Joy contributors

### PR DESCRIPTION
**Pull request by pm0kjp**

### CODE REVIEW

File: _for_authors/module_updates.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+### Additional Notes
```
Comment (rosemm) [inline _for_authors/module_updates.md:25]:
```suggestion
## Additional Notes 
```

File: _for_authors/module_updates.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+This need for updating and maintenance on an ongoing basis can prove problematic for modules contributed by authors who are not core staff who work together daily.  You may be too busy to make suggested changes, or your interests may have changed and you're just not interested in continuing to donate your labor to continual improvement.  We may not have the most up-to-date contact information for you and might not even be able to reach you to inform you about desired changes.
+
+## Updates We Initiate
```
Comment (rosemm) [inline _for_authors/module_updates.md:9]:
```suggestion
## Updates You Initiate

If at any point you wish to suggest a change to a module you wrote, please let us know! You can send us an email at dart@chop.edu, or [post an issue](https://github.com/arcus/education_modules/issues/new/choose) to let us know what should be changed. 

We will take your suggestions into account and use them at our discretion.  If at any point the module diverges from your vision and you prefer that the attribution be changed, you can request us to change the attribution to be a co-authorship between you and DART staff or ask us to make this attribution a generic "DART contributor" attribution and remove your name from the module. 

## Updates We Initiate
```

Comment (rosemm) [reply _for_authors/module_updates.md:9]: In a meeting we talked about including a brief section on author-initiated changes as well, to make it clear we welcome that.

**Review by rosemm — CHANGES_REQUESTED** [review]
I suggested a few tweaks based on my (fuzzy?) memories from a meeting when we discussed this. Take or leave as you see fit.

**Review by rosemm — CHANGES_REQUESTED** [review]
I forgot --- there were also a few additional notes from other sources (QA templates, module template) that should be integrated into the inclusivity guidelines.

**Comment by pm0kjp** [comment]
@rosemm fantastic and useful suggestions, I committed them all verbatim.

**Review by rosemm — APPROVED** [review]

**Pull request merged**